packages/host/app/components/operator-mode/code-submode.gts

```diff
@@ -252,6 +256,37 @@ export default class CodeSubmode extends Component<Signature> {
     return !this.codePath || this.currentOpenFile?.state === 'not-found';
   }
 
+  private get rhsIncompatibleMessage() {
```

Comment:
can we choose a more descriptive name than "rhs*" naming, for example "fileIncompatibilityMessage"?